### PR TITLE
update charts for OCI

### PR DESCRIPTION
**Pull request by sbtaylor15**
## Description

Updated the README.md and the storage yaml files to handle deploying to OCI.  FYI, Oracle folks are going to update the installers/helm/pyrsia-node/README.md with the necessary oci command line steps in a separate PR.

Fixes pyrsia/pyrsia#996

To test this PR a deployment to the OCI cluster must be performed.  Oracle folks will perform this testing.

## PR Checklist

- [X] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [X] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/get_involved/good_pr.md)
- [X] I've included a good title and brief description along with how to review them.
- [X] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).

### Code Contributions

- [X] I've built the code `cargo build --all-targets` successfully.
- [X] I've run the unit tests `cargo test --workspace` and everything passes.
- [X] I've made sure my rust toolchain is current `rustup update`.

### CODE REVIEW

File: installers/helm/pyrsia-node/README.md [hunk 1]
```diff
@@ -43,14 +58,18 @@ The command deploys Pyrsia on the Kubernetes cluster using the following paramet
        gcloud projects add-iam-policy-binding <PROJECT> --member=serviceAccount:service-<PROJECT_NUMBER>@compute-system.iam.gserviceaccount.com --role=roles/cloudkms.cryptoKeyEncrypterDecrypter
        ```
 
+  - Oracle
+    1. Step One
```
Comment (efrisch) [inline installers/helm/pyrsia-node/README.md]: Can we elaborate on step one and 2?

**Review by efrisch — APPROVED** [review]
👍

File: installers/helm/pyrsia-node/README.md (hunk 1 above)
Comment (sbtaylor15) [reply installers/helm/pyrsia-node/README.md]: Added placeholder wording

**Review by fishseabowl — APPROVED** [review]
lgtm

**Pull request merged**